manifests/m/Mozilla/Firefox/12.0/Mozilla.Firefox.locale.en-US.yaml

```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mozilla.Firefox
+PackageVersion: 12.0
+PackageLocale: en-US
+Publisher: Mozilla
+PackageName: Firefox 12
+License: Mozilla
+ShortDescription: Firefox 12 WIN 32

```

Comment:
ShortDescription: a free web browser backed by Mozilla, a non-profit dedicated to internet health and privacy.